pom.xml

```diff
@@ -53,7 +53,7 @@
         <!-- At least one "structures" JAR must also be included -->
         <dependency>
             <groupId>ca.uhn.hapi.fhir</groupId>
-            <artifactId>hapi-fhir-structures-dstu3</artifactId>
```

Comment:
@jamesagnew can you confirm if this change is needed? I've built jpaserver-starter for R4 without changing this in the pom.xml before...

Reply:
you need the corresponding structures package to the fhir_version in the properties file, otherwise you will get runtime errors.

Reply:
Oddly, I have not experienced that... I have an R4 jpaserver-starter server running right now that hasn't complained about the structures be the wrong version. I hadn't noticed this line in pom.xml, so I didn't change it when I built my R4 server...

Reply:
you are right, all fhir versions are already included in jpaserver-base, import was unnecessary --> removed